### DISCUSSION TITLE
在淘宝小程序中，导入HDR文件之后，页面出现卡顿，模型缩放、旋转也卡

### DISCUSSION THREAD

**Discussion by flyingflyfly** [galacean/discussion#12] [Mini Program（小程序）]
1、HDR文件：http://next-ubanner.oss-cn-beijing.aliyuncs.com/AE_BODY_MOVIN/TMP/9195088dfe33208b9b92faf56ebe55a3/hdr.hdr.env
2、模型：https://next-ubanner.oss-cn-beijing.aliyuncs.com/AE_BODY_MOVIN/TMP/2afcc40b678752f3c547a13124f7bef2/0002.gltf

**Comment by singlecoder** [comment]
你代码的使用方式提供下，最好有最小工程示例

**Reply by flyingflyfly** [reply]
实测0.6版本没有该现象，0.7、0.8都有卡顿现象
